### PR TITLE
Rss feed and fly bys

### PR DESCRIPTION
**Pull request by lawand7**
XMLParser for RssFeeds and ISSFlybys

### CODE REVIEW

File: Backend/Tools/ISSFlybysXMLParser.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+from xml.etree.ElementTree import Element
+from xml.etree.ElementTree import tostring
+from Backend.Core.dataStructs import ISSDBKey
+
+
+# XML for ISSFlyBys
+# # <Request>
+# #	<requestName>ISSFlyBys</requestName>
```
Comment (lkno0705) [inline Backend/Tools/ISSFlybysXMLParser.py:8]: The request name is wrong. A country flyby != a past flyby of the users current location! You can't just call it flybys. Change this to 2 function which call your main function.

File: Backend/Tools/rssFeedXMLParser.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+from xml.etree.ElementTree import Element
```
Comment (lkno0705) [inline Backend/Tools/rssFeedXMLParser.py:1]: Why is this file called rssFeedXMLParser if it doesn't use rssFeed data?

Comment (lawand7) [reply Backend/Tools/rssFeedXMLParser.py:1]: this function will be moved to the main XMLParser, where rssFeed Data is given as requestData through function reformatData

Comment (lawand7) [reply Backend/Tools/rssFeedXMLParser.py:1]: after I move this, I'll add all needed imports

**Pull request merged**